client/src/services/User.js

```diff
@@ -47,7 +47,7 @@ export function handleLogin(data, history) {
         email: json.user.email,
         first_name: json.user.profile.first_name,
         image: json.user.profile.image
-      });
+      }, getUserData);
```

Comment:
Are you chaining this method here? It would be better for the login endpoint to just return the data we need here rather than make a separate API call afterwards

Reply:
done